webservices/partition/base.py

```diff
@@ -316,6 +277,7 @@ def refresh_child(cls, cycle, queue_old, queue_new):
             # populating the child tables during normal partitioning.
             # Otherwise, an error is thrown due more values being specified
             # than there are columns to accept them.
+            # VM TODO - Is this still needed?
```

Comment:
I thought about this some more and I think it is still needed.  The queue tables aren't going away with these changes, and they're setup with the `two_year_transaction_period` column in them.  By having that column be a part of the record, the column processing that takes place in here will run into a conflict and halt because it will expect it to not be there.

I'm sorry if this is confusing, I'm happy to cover in more detail if you'd like!